src/features/ClinicDashboard/services/helpers.ts

```diff
@@ -1,6 +1,6 @@
 import moment from 'moment';
 
-export const getTimeOfVisit = (visit_date) => {
+export const getTimeOfVisit = (visit_date:string) => {
```

Comment:
название переменных camelCase
```suggestion
export const getTimeOfVisit = (visitDate:string) => {
```